src/index.js

```diff
@@ -1,107 +1,54 @@
-      } else if (file.displayName === "FileRemote" && tabularFormats.includes(file.descriptor.format)) {
-        // Tabular data
-        try {
-          const rowStream = await file.rows({size: 100, keyed: true})
-          const data = await toArray(rowStream)
-          if (data.length > 0) {
-            file.descriptor.data = data // This makes it FileInline
-          } else {
-            file.descriptor.unavailable = true
-          }
-        } catch (e) {
-          console.log(e)
-          file.descriptor.unavailable = true
-        }
-      } else if (file.descriptor.format.toLowerCase().includes('json')) {
-        // Geographical data
-        const response = await fetch(file.descriptor.path)
-        if (!response.ok) {
-          file.descriptor.unavailable = true
-          return
+const datapackage = {
+  views: [
+    {
+      "id": 1,
+      "resources": [
+        {
+          "name": "test",
+          "data": [
+            [
+              "a",
+              "b",
+              "c",
+              "d"
+            ],
+            [
+              1,
+              2,
+              3,
+              4
+            ],
+            [
+              2,
+              3,
+              4,
+              5
+            ],
+            [
+              3,
+              4,
+              5,
+              6
+            ]
+          ]
         }
-        const result = await response.json()
-        // The '.json' files can contain geo data - check by its 'type' property
-        const geoJsonTypes = [
-          'Feature', 'FeatureCollection', 'Point', 'MultiPoint', 'LineString',
-          'MultiLineString', 'Polygon', 'MultiPolygon', 'GeometryCollection'
-        ]
-        if (geoJsonTypes.includes(result.type)) {
-          file.descriptor.data = result
-        } else {
-          // It isn't a valid GeoJSON
-          file.descriptor.unavailable = true
-          return
-        }
-      } else if (file.descriptor.format.toLowerCase() === 'pdf') {
-        return
-      } else {
-        // We can't load any other data types for now.
-        file.descriptor.unavailable = true
-      }
-
-      // Compile views and render App. Calling this here so components are
-      // re-rendered after each loop so users can see a preview as soon as
-      // it's available. Useful when you have lots of views to render.
-      render(dataset.descriptor)
-    })
-
-    // Compile views and render App
-    render(dataset.descriptor)
-  })
-  .catch((error) => {
-    console.warn('Failed to load a Dataset from provided datapackage id\n' + error)
-  })
+      ],
+      "specType": "table"
+    }
+  ]
 }
 
-function render(descriptor) {
-  // Compile views and render App
-  descriptor.views.forEach(view => {
-    const compiledView = dpRender.compileView(view, descriptor)
-    ReactDOM.render(
-      <App view={compiledView} />,
-      document.getElementById(`datapackage-view-${view.id}`)
-    )
-  })
-}
+ReactDOM.render(
```

Comment:
I don't think we need this either. I think we just need:
```javascript
export default App
```
so that we can load this in the parent app (Data Explorer, for instance) like this:

```javascript
import FilterUI from './components/FilterUI'
import DataView from 'datapackage-views'
import DataViewBuilder from './components/DataViewBuilder'
import { filterUIAction, fetchDataAction, dataViewBuilderAction } from './actions/';

export const App = props => {
  
  return (
    <div className="text-center ml-6">
      <header>
        <div className="container">
          <h1 className="text-3xl">Data Explorer</h1>
        </div>
      </header>
      <div className="container py-6 bg-pink-100">
        <div className="">
          <FilterUI {...props} />
        </div>
      </div>
      <div className="container flex py-6 bg-yellow-100">
        <div className="w-3/4 p-4 mr-4 overflow-x-auto">
          <DataView {...props} />
        </div>
        <div className="w-1/4 p-4 mr-4 bg-green-100">
          <DataViewBuilder {...props} />
        </div>
      </div>
     </div>
  )
}
```

We might then need to add a wrapper that includes this App for Datahub.io or Preview pages, but I think the module should just export the component.